### PR TITLE
[#307] Mask client secret in CLI config

### LINKED ISSUE #307: not masked ODAHUFLOWCTL_OAUTH_CLIENT_SECRET configuration variable value
**Prerequisite**
installed odahuflowctl

**To Reproduce**
1. login through odahuflowctl by using --url, --client_id, --client_secret, --issuer flags
2. type odahuflowctl config all
3. and you can see that ODAHUFLOWCTL_OAUTH_CLIENT_SECRET value is not masked

**Expected behavior**
ODAHUFLOWCTL_OAUTH_CLIENT_SECRET variable value should be masked

**Actual behavior**
ODAHUFLOWCTL_OAUTH_CLIENT_SECRET variable value is not masked

**Environment:**
 - OS: macOS 10.15.5
 - cli, sdk versions: 1.2.0-rc7

### PR DESCRIPTION
**Pull request by keshamin**
Fix #307 
Mask `ODAHUFLOWCTL_OAUTH_CLIENT_SECRET` (and other properties ending with `_SECRET` as well) in output of `odahuflowctl config all`.

Before:
```
ODAHUFLOWCTL_OAUTH_CLIENT_SECRET - Set OAuth2 Client id
  default: ''
  current: 'abc'
```

After:
```
ODAHUFLOWCTL_OAUTH_CLIENT_SECRET - Set OAuth2 Client id
  default: ''
  current: '****'
```

### CODE REVIEW

**Review by BPylypenko — APPROVED** [review]

**Review by easokol — APPROVED** [review]

**Review by vlad-tokarev — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -6,8 +6,7 @@ services:
 install: skip
 branches:
   only:
-  - master
```
Comment (karbyshevds) [inline .travis.yml:9]: Please remove this unrelated changes.

**Review by karbyshevds — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (keshamin) [reply .travis.yml:9]: Yeah, sure, it anyway will force me to do that because of a merge conflict :)
